Change files

mikeelrod opened the pull request:
File management refined.

mikeelrod committed:
Change files

File management refined.

michaelja2 commented on Webspage 3.0/index.htm:
```diff
@@ -1,15 +1,21 @@
-<h1>Mike Elrod</h1>
-<h2>Portfolio</h2>
 
+	<h1>MIKE ELROD</h1>
```
Excellent. You will want an H1 on every page.  It tells search engines what the page is about and helps SEO.

michaelja2 commented on Webspage 3.0/index.htm:
```diff
@@ -1,15 +1,21 @@
+		video
+		graphics
+		photo
+    </div>
```
Keep your indentations consistent. When you have a lot of code on the page, you need to be able to scan it easily.

michaelja2 commented on Webspage 3.0/styles/main.css:
```diff
@@ -0,0 +1,58 @@
+h1 {
+	text-align: left;
+	font-family: roboto;
```
You will want to have a fallback, in case someone does not have roboto. You will want to change this to-> 

`font-family: roboto, sans-serif;`

michaelja2 replied:
They way this works is that they try the first font. If the user does not have it, the move to the next, and then the next. You can have as many as you want, but end with serif or sans-serif (defaults)

michaelja2 commented on Webspage 3.0/styles/main.css:
```diff
@@ -0,0 +1,58 @@
+	line-height: .5;
+
+}
+
```
You will want to add a body style. The body is the full group of visible content on a page. This will prevent you from having to repeat common styles. Then you can just override styles on specific elements. RE:

`body {
  font-family: roboto, sans-serif;
  font-variant: thin;
}`

michaelja2 reviewed (approved):
Overall, looks good. Just a few changes to save you time. You might want to focus on one page, then copy out to the others, so you don't have to update all the pages every time you make a bigger change. 

Keep going man!

michaelja2 committed:
Create README

michaelja2 committed:
example of links and padding

mikeelrod committed:
Spacing and links

Spacing/padding has been added as well as text descriptions on each page. Menu has been incorporated into each page. Still needs a home button for familiarity for audience.

Also, the links don't seem to resolve to the pages correctly. Not sure what to do here.

michaelja2 commented on Webspage 3.0/graphics.htm:
```diff
@@ -0,0 +1,31 @@
+<html>
+<head>
+<link rel="stylesheet" type="text/css" href="styles/main.css">
+</head>
```
You may also want to add a <title> tag in the head. This is what shows up in the tab/browser.

michaelja2 commented on Webspage 3.0/index.htm:
```diff
@@ -1,15 +1,23 @@
-<link rel="stylesheet" type="text/css" href="mystyles.css">
-</head>
+	<head>
+		<link rel="stylesheet" type="text/css" href="styles/main.css">
```
nice
